### PR TITLE
Updating readme with CLI example and footer note

### PR DESCRIPTION
**Pull request by totus**
@ekondrashev please review & LGTM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,4 +27,6 @@ The program expects target url to take HTML(and check for dead links) from.
 }
 ```
 
-It is ok to use YML/XML or any other format considering the test verifying this format is availabl
+It is ok to use YML/XML or any other format (not only JSON) considering the test verifying this format is available.
+
+*N.B.: it worth to think of various edge cases related to CLI usage, potential connection and/or content issues and cover them in your code*
```
Comment (ekondrashev) [inline README.md]: @totus edge cases are good candidates for further(2, 3, etc) steps. Lets not make initial task too complex.

Comment (ekondrashev) [reply README.md]: So the suggestion is to skip 32th line for now. The rest looks ok!

**Comment by ekondrashev** [comment]
Lets move this part to `phase2`:
```
*N.B.: it worth to think of various edge cases related to CLI usage, potential connection and/or content issues and cover them in your code*
```
